src/main/resources/messages.properties

```diff
@@ -130,6 +130,9 @@ validation.element.missing.creditors_after_one_year.previous_period.total = Ente
 validation.unexpected.data.creditors_after_one_year.current_period = The total figure here must match the 'Creditors - amounts falling due after one year' figure on the balance sheet
 validation.unexpected.data.creditors_after_one_year.previous_period = The total figure here must match the 'Creditors - amounts falling due after one year' figure on the balance sheet
 
+#Fixed assets investments
+validation.characters.invalid.fixed_assets_investments.details = Invalid characters entered
+validation.length.invalidInputLength.fixed_assets_investments.details = Field must not exceed 20,000 characters
```

Comment:
As a result of the api validation, this message should read `Field must not exceed {1} characters`. The validation mapping utility in the web will then read index 1 of the `Error` object's arguments, which will be 20000. Better to have a placeholder here in case the upper limit on the length of the string ever changes.

Reply:
Updated to use the placeholder